### PR TITLE
Fix routing after login

### PR DESCRIPTION
**Pull request by egaluza**

### CODE REVIEW

File: packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/auth.service.ts [hunk 1]
```diff
@@ -3,34 +3,35 @@ import { CanActivate, Router, ActivatedRouteSnapshot } from '@angular/router';
 
 @Injectable()
 export class AuthService {
-  loggedIn = true;
+    loggedIn = true;
+    prevPath: string;
```
Comment (nightskylark) [inline packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/auth.service.ts:7]: Bad field name

Comment (nightskylark) [reply packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/auth.service.ts:7]: And where is default value?

Comment (egaluza) [reply packages/devextreme-schematics/src/add-layout/files/src/app/shared/services/auth.service.ts]: Fixed. '/' is default value

**Comment by nightskylark** [comment]
It looks like the indent should have two spaces.
https://github.com/DevExpress/devextreme-angular-template/blob/482c89a27e47ebd317df4971ddf48a6ac17032b5/.editorconfig#L7

**Pull request merged**